src/Swarm/TUI/Model.hs

```diff
@@ -808,8 +818,16 @@ initAppState userSeed scenarioName toRun cheatMode = do
   case skipMenu of
     False -> return $ AppState gs ui
     True -> do
-      scenario <- loadScenario (fromMaybe "classic" scenarioName) (gs ^. entityMap)
-      liftIO $ execStateT (scenarioToAppState scenario userSeed toRun) (AppState gs ui)
+      (scenario, path) <- loadScenario (fromMaybe "classic" scenarioName) (gs ^. entityMap)
+      as <- liftIO $ execStateT (scenarioToAppState scenario userSeed toRun) (AppState gs ui)
+      normalPath <- liftIO $ normalizeScenarioPath (as ^. gameState . scenarios) path
+      liftIO . appendFile "/tmp/debug" $ "TODO: M - Scenario in progress " <> normalPath <> "\n"
+      liftIO . appendFile "/tmp/debug" $ "TODO: M - Originally " <> path <> "\n"
+      t <- liftIO getZonedTime
+      return $
+        as
+          & gameState . currentScenarioPath ?~ normalPath
+          & gameState . scenarios . scenarioItemByPath normalPath . _SISingle . _2 . scenarioStatus .~ InProgress t 0
```

Comment:
This code definitely shouldn't be duplicated both here and in `startGame`.  Can we just call `startGame` here?